### ISSUE TITLE
Cutoff dropdowns on Manage Events tab still show archaic formats post 9b2

### ISSUE THREAD

**Issue by weatherman223** [thewca/worldcubeassociation.org#7555]
On the WCA site, you are still able to select "best of 2" under the 6x6 and 7x7 events when adding a cutoff to them in the Manage Events screen. Bo2 is not a legal cutoff under article 9b2. 

See here for screenshot. 

![image](https://user-images.githubusercontent.com/36021082/213900561-b3a2f4ae-f99b-4c4a-839d-ee39239319b6.png)

An example of this being on a competition website can be viewed with Pioneer Valley Cubing B 2023 - https://www.worldcubeassociation.org/competitions/PioneerValleyCubingB2023#competition-schedule

![image](https://user-images.githubusercontent.com/36021082/213900584-7373eac8-b6ff-466f-b8d0-0f999960eacc.png)

**Comment by FinnIckler** [comment]
This has been in the works for a year.. https://github.com/thewca/worldcubeassociation.org/pull/6593

**Comment by dunkOnIT** [comment]
Left open as #6593 doesn't seem to reference a specific issue. @dmint789 would you be able to give feedback on whether you could push the PR to a close?

**Comment by dunkOnIT** [comment]
@kr-matthews Thanks for the reference to this issue. It seems like there are two levels we could fix this at: 
1. In the Rails modelling (by better defining the available cutoff formats for each event) 
2. In the frontend (by just not showing certain invalid cutoffs)

I definitely prefer Option 1, and the fix doesn't seem _that_ hard, but I'm not sure we have the Rails resources to put on it until the Registration system is live. 

**How hard do you think Option 2 would be to implement?** And how much do you hate the "hackiness" of it? Given that this is causing real inconvenience/confusion for organizers, and the fix seems relatively simple, it would be nice to at least pursue a hotfix.

**Comment by kr-matthews** [comment]
I think it would be quite easy to just explicitly filter out bo2 for 6x6x6 and 7x7x7 in the front-end. (Those are the only 2 incorrect ones right?) Once a proper back-end fix is merged then the temporary front-end fix commit could be reverted.

As you say, it's a bit of a hack, and things that are meant to be temporary have a tendency to stick around a long time in software... But if @gregorbg is ok with this then I can put up a PR. I agree that it would be best to stop organizers/delegates from accidentally selecting these options.

**Comment by gregorbg** [comment]
Yup, your suggestion of hard-coding 666 and 777 seems like the most efficient way forward.
